evap/contributor/forms.py

```diff
@@ -44,23 +44,26 @@ def clean(self):
 
         vote_start_datetime = self.cleaned_data.get('vote_start_datetime')
         vote_end_date = self.cleaned_data.get('vote_end_date')
+
+        # IMPORTANT: when removing following feature (auto-set a vote_start_datetime of the past to the future),
+        # make sure to alter the according test ("start date in the past -> form invalid?")!
+        # Location of test: evap/staff/tests/test_forms.py
+        #                   EvaluationFormTests.test_contributor_evaluation_form_date_validation(self):
+        if vote_start_datetime and vote_start_datetime < datetime.now():
+            self.cleaned_data['vote_start_datetime'] = round_datetime_up(round_to=5 * 60)
+
         if vote_start_datetime and vote_end_date:
             if vote_start_datetime.date() > vote_end_date:
                 self.add_error("vote_start_datetime", "")
                 self.add_error("vote_end_date", _("The first day of evaluation must be before the last one."))
 
-    def clean_vote_start_datetime(self):
-        vote_start_datetime = self.cleaned_data.get('vote_start_datetime')
-        if vote_start_datetime and vote_start_datetime < datetime.now():
-            raise forms.ValidationError(_("The first day of evaluation must be in the future."))
-        return vote_start_datetime
-
     def clean_vote_end_date(self):
-        vote_end_date = self.cleaned_data.get('vote_end_date')
+        vote_end_date = self.cleaned_data['vote_end_date']
 
         # The actual deadline is EVALUATION_END_OFFSET_HOURS:00 AM of the day after vote_end_date.
         # Therefore an evaluation date 24h + EVALUATION_END_OFFSET_HOURS in the past would technically still be in the future.
-        if vote_end_date and date_to_datetime(vote_end_date) + timedelta(hours=24 + settings.EVALUATION_END_OFFSET_HOURS) < datetime.now():
+        if vote_end_date and date_to_datetime(vote_end_date) + timedelta(hours=24 +
+           settings.EVALUATION_END_OFFSET_HOURS) < datetime.now():
```

Comment:
I dislike the three spaces indent, tbh. I'd make it an early out
```
if not vote_end_date:
    return vote_end_date
```
directly at the start and then below the condition would be shorter:
```
if date_to_datetime(vote_end_date) + timedelta(...) < datetime.now():
```
Maybe put the calculation of the timedelta in a new line. Actually, I think there should be one central place where vote_end datetimes are calculated from vote_end_dates anyway, so I'd rather like this
```
end_datetime = Evaluation.calculate_end_datetime_from_end_date(end_date) # naming to be done
if end_datetime < datetime.now():
   raise ...
```